src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -388,7 +388,26 @@
    writeLines(lines, odbcinstPath)
 })
 
-.rs.addFunction("odbcBundleRegisterLinux", function(name, driverPath, version) {
+.rs.addFunction("odbcOdbcInstLibPath", function() {
+   odbcinstLib <- NULL
+
+   odbcinstBin <- system2("which", "odbcinst", stdout = TRUE)
+   if (is.null(odbcinstBin) || length(odbcinstBin) == 0) {
```

Comment:
This is redundant (`length(NULL)` is 0) -- was this meant to be `nchar`/`nzchar`?

Reply:
`system2()` returns a `character(0)` so I do need `length(odbcinstBin)`, otherwise `nchar(character(0))` would return `integer(0)` . and break the if statement, but I can remove `is.null(odbcinstBin)`, removed.